log4j-jndi/src/main/java/org/apache/logging/log4j/jndi/JndiManager.java

```diff
@@ -194,6 +195,14 @@ protected boolean releaseSub(final long timeout, final TimeUnit timeUnit) {
         return JndiCloser.closeSilently(this.context);
     }
 
+    /**
+     * @return the active context
+     */
+    @Nullable
+    public Context getContext() {
+        return context;
+    }
```

Comment:
I have mixed feeling about exposing `Context` (which is usually `InitialContext`, the source of all evil) from the public API.